src/configs/axios.config.ts

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+export const axiosClient = axios.create({
+  baseURL: 'http://localhost:5000/',
```

Comment:
I think it's better to have baseURL in the .env

Reply:
Agreed, move to env